### PR TITLE
autoexport: Add support for console exporters

### PR DESCRIPTION
**Pull request by codeboten**
This adds support for the standard output exporter for traces and metrics. To remain consistent with other implementations, i used the key `console` as the identifier for the exporter. This change is related to the specification pull request https://github.com/open-telemetry/opentelemetry-specification/pull/3742

### CODE REVIEW

File: exporters/autoexport/metrics.go [hunk 1]
```diff
@@ -38,6 +39,7 @@ func WithFallbackMetricReader(exporter metric.Reader) MetricOption {
 // OTEL_METRICS_EXPORTER defines the metrics exporter; supported values:
 //   - "none" - "no operation" exporter
 //   - "otlp" (default) - OTLP exporter; see [go.opentelemetry.io/otel/exporters/otlp/otlpmetric]
+//   - "console" - Standard output exporter; see [https://pkg.go.dev/go.opentelemetry.io/otel/exporters/stdout/stdoutmetric]
```
Comment (pellared) [inline exporters/autoexport/metrics.go:42]:
```suggestion
//   - "console" - Standard output exporter; see [go.opentelemetry.io/otel/exporters/stdout/stdoutmetric]
```

File: exporters/autoexport/spans.go [hunk 2]
```diff
@@ -43,6 +44,7 @@ func WithFallbackSpanExporter(exporter trace.SpanExporter) SpanOption {
 // OTEL_TRACES_EXPORTER defines the traces exporter; supported values:
 //   - "none" - "no operation" exporter
 //   - "otlp" (default) - OTLP exporter; see [go.opentelemetry.io/otel/exporters/otlp/otlptrace]
+//   - "console" - Standard output exporter; see [https://pkg.go.dev/go.opentelemetry.io/otel/exporters/stdout/stdouttrace]
```
Comment (pellared) [inline exporters/autoexport/spans.go:47]:
```suggestion
//   - "console" - Standard output exporter; see [go.opentelemetry.io/otel/exporters/stdout/stdouttrace]
```

**Review by pellared — COMMENTED** [review]
https://github.com/open-telemetry/opentelemetry-specification/pull/3742: it would be good to make sure there is an agreement on the "name" before merging 😉

**Review by codeboten — COMMENTED** [review]
Thanks for reviewing @pellared, I agree that having a consistent name would be good :) This is at least consistent w/ 3 (python, js, ruby) implementations. Java uses the term `logging` https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#logging-exporter which I worry is causing the same confusion as the `logging` exporter did in the Collector before it was renamed to `debug`

**Review by pellared — APPROVED** [review]
> This is at least consistent w/ 3 (python, js, ruby) implementations. 

I suggest calling it out in https://github.com/open-telemetry/opentelemetry-specification/pull/3742

**Review by MrAlias — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `"go.opentelemetry.io/contrib/config"` package that includes configuration models generated via go-jsonschema. (#4376)
 - Add `NewSDK` function to `"go.opentelemetry.io/contrib/config"`. The initial implementation only returns noop providers. (#4414)
 - Add metrics support (No-op, OTLP and Prometheus) to `go.opentelemetry.io/contrib/exporters/autoexport`. (#4229, #4479)
+- Add support for standard output exporter via environment variables for the metric and trace signals identified by the key: `console`. (#4486)
```
Comment (pellared) [inline CHANGELOG.md:18]:
```suggestion
- Add support for `console` span exporter and metrics exporter in `go.opentelemetry.io/contrib/exporters/autoexport`. (#4486)
```

**Pull request merged**